### PR TITLE
add missing parameters in enable mixer script

### PR DESCRIPTION
**Pull request by maplebeats**
Re-enable Mixer bash script not work now , it lacks the filename parameters。

### CODE REVIEW

File: perf/benchmark/README.md [hunk 1]
```diff
@@ -162,7 +162,7 @@ Calls to Istio's Mixer component (policy and telemetry) adds latency to the side
 
     ```bash
     kubectl -n istio-system get cm istio -o yaml > /tmp/meshconfig.yaml
-    python ../../bin/update_mesh_config.py enable_mixer /tmp/meshconfig.yaml  | kubectl -n istio-system apply -
+    python ../../bin/update_mesh_config.py enable_mixer /tmp/meshconfig.yaml  | kubectl -n istio-system apply -f /tmp/meshconfig.yaml
```
Comment (howardjohn) [inline perf/benchmark/README.md]: this doesn't look right, why are you piping to kubectl but then applying a file?

Comment (howardjohn) [reply perf/benchmark/README.md]: Should this be `kubectl -n istio-system apply -f -`?

Comment (maplebeats) [reply perf/benchmark/README.md]: > 
> 
> Should this be `kubectl -n istio-system apply -f -`?

I thought the disable_mixer was working normally, so copy that command, a closer look found that the script is also abnormal , I fix it.

**Comment by maplebeats** [comment]
@

> 
> 
> @maplebeats Please verify your changes works. Like after you disable mixer, then re-enable it, the re-enable works.

fixed, the disable script didn't not work either。

**Pull request merged**